### PR TITLE
Swift 3 support

### PR DESCRIPTION
**Pull request by brockboland**
Updated the Swift sample projects to use Swift 3. I think the template for the Cat2Cat output is fine how it is, but I'm curious to see if Travis has any issues running this.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: objective-c
-osx_image: xcode7
+osx_image: xcode8
```
Comment (designatednerd) [inline .travis.yml]: do they not have an 8.1 image yet? Or is this purposely 8 so that it uses 3.0 instead of 3.0.1?

Comment (designatednerd) [inline .travis.yml]: Yeah i don't think it changed anything that wasn't already a little crazy, which I think we tried not to be here.

**Comment by brockboland** [comment]
Looks like I need to switch this one to use `scan` for tests

**Comment by brockboland** [comment]
There we go. @vokal/ios-developers @designatednerd review please?

**Pull request merged**